test/functional/content_view/version/update_test.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../../test_helper'
+require 'hammer_cli_katello/content_view_version'
+
+module HammerCLIKatello
+  describe ContentViewVersion::UpdateCommand do
+    include OrganizationHelpers
+    it 'allows minimal options' do
+      api_expects(:content_view_versions, :update)
```

Comment:
`.with_options(id: 2, description: 'pizza')` or similar

Reply:
ignore me it is with params me--

Reply:
whoops, sorry for the typo! :)